model/trace/messaging.yaml

```diff
@@ -15,16 +15,16 @@ groups:
           The [conversation ID](#conversations) identifying the conversation to which the message belongs,
           represented as a string. Sometimes called "Correlation ID".
         examples: 'MyConversationId'
-      - id: message.payload_size_bytes
+      - id: message.envelope.size
         type: int
         brief: >
-          The (uncompressed) size of the message payload in bytes.
-          Also use this attribute if it is unknown whether the compressed or uncompressed payload size is reported.
+          The size of the uncompressed message body and metadata in bytes.
```

Comment:
see my comment below, I think we have no way of knowing whether the body is compressed (when done by application code). 

I'd prefer to leave it a bit vague allowing specific extensions to document what makes the most sense for them.